test/extended/olm/olmv1.go

```diff
@@ -0,0 +1,114 @@
+package operators
+
+import (
+	"encoding/json"
+	"path/filepath"
+	"time"
+
+	g "github.com/onsi/ginkgo/v2"
+	o "github.com/onsi/gomega"
+	e2e "k8s.io/kubernetes/test/e2e/framework"
+
+	exutil "github.com/openshift/origin/test/extended/util"
+	apimeta "k8s.io/apimachinery/pkg/api/meta"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+var _ = g.Describe("[sig-operator] OLM V1", func() {
+	defer g.GinkgoRecover()
+
+	var (
+		oc = exutil.NewCLI("default")
+
+		buildPruningBaseDir = exutil.FixturePath("testdata", "olm")
+		catalog             = filepath.Join(buildPruningBaseDir, "catalog.yaml")
+		operator            = filepath.Join(buildPruningBaseDir, "operator.yaml")
+	)
+
+	//TODO remove this check once OLM V1 has graduated
+	if !exutil.IsTechPreviewNoUpgrade(oc) {
+		g.Skip("the test is not expected to work within Tech Preview disabled clusters")
```

Comment:
Nit: double negative

```suggestion
		g.Skip("the test is only expected to work within Tech Preview clusters")
```